keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -73,279 +75,320 @@
 
 Here is a TL;DR version:
 
- - BucketClaims/Bucket are similar to PVC/PV. 
- - BucketClaim is used to request generation of new buckets. 
- - Buckets represent the actual Bucket. 
- - BucketClass is similar to StorageClass. It is meant for admins to define and control policies for Bucket Creation
- - BucketAccess is required before a bucket can be "attached" to a pod.
- - BucketAccess both represents the attachment status and holds a pointer to the access credentials secret.
- - BucketAccessClass is meant for admins to control authz/authn for users requesting access to buckets.
+- BucketClaims/Bucket are similar to PVC/PV.
+- BucketClaim is used to request generation of new buckets.
+- Buckets represent the actual Bucket.
+- BucketClass is similar to StorageClass. It is meant for admins to define and control policies for Bucket Creation
+- BucketAccess is required before a bucket can be "attached" to a pod.
+- BucketAccess both represents the attachment status and holds a pointer to the access credentials secret.
+- BucketAccessClass is meant for admins to control authz/authn for users requesting access to buckets.
 
-## Bucket Creation
+### Bucket Creation
 
 The following stakeholders are involved in the lifecycle of bucket creation:
 
- - Users  - request buckets to be created for their workload
- - Admins - setup COSI drivers in the cluster, and specify their configurations
+- Users  - request buckets to be created for their workload
+- Admins - setup COSI drivers in the cluster, and specify their configurations
 
 Here are the steps for creating a Bucket:
 
-###### 1. Admin creates BucketClass, User requests Bucket to be created using BucketClaim
-
-The BucketClass represents a set of common properties shared by multiple buckets. It is used to specify the driver for creating Buckets, and also for configuring driver-specific parameters. More information about BucketClass is [here](#bucketclass)
-
-The BucketClaim is a claim to create a new Bucket. This resource can be used to specify the bucketClassName, which in-turn includes configuration pertinent to bucket creation. More information about BucketClaim is available [here](#bucketclaim)
-
-```
-    BucketClaim - bcl1                                    BucketClass - bc1
-    |------------------------------|                      |--------------------------------|
-    | metadata:                    |                      | deletionPolicy: delete         |
-    |   namespace: ns1             |                      | driverName: s3.amazonaws.com   |
-    | spec:                        |                      | parameters:                    |
-    |   bucketClassName: bc1       |                      |   key: value                   |
-    |   protocols:                 |                      |--------------------------------|
-    |   - s3                       |
-    |------------------------------|
-                                                          
-``` 
```

Comment:
These block-style things seem like they must be created with a markdown tool that I don't know about. Since this is a non-standard tool, I am also going to replace all instances of these with simpler built-in markdown blocks like this:

```
    ```yaml
    apiVersion: objectstorage.k8s.io/v1alpha2
    kind: BucketClass
    metadata:
      name: silver
    spec:
      deletionPolicy: delete
      driverName: s3.amazonaws.com
      parameters:
        key: value
    ```
```